var.get() function does not throws an error if the asked value does not exist in the dictionary.
But var[] throws an error if the asked value does not exist in dictionary.
Dictionary is ordered after python 3.7 
